src/test/java/decodex/storage/StorageTest.java

```diff
@@ -0,0 +1,25 @@
+package decodex.storage;
+
+import decodex.data.exception.FileException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class StorageTest {
+    
+    @Test
+    void readFromDefaultInputFile_inputFileWithSomeContent_stringContent() throws FileException {
```

Comment:
Unnecessary files should not be committed and pushed. We might need to figure out how to test the functionalities without dummy files.